### PR TITLE
Add, view and filter tips

### PR DESCRIPTION
**Pull request by katbow**
Currently this contains a section to input tips with tags, however this may be removed/only accessed by certain people down the road. 

I kept it in for now while I wait to hear how the tips should be input.

### CODE REVIEW

**Comment by katbow** [comment]
@nelsonic I added the feature list for story content that I missed in the last PR here. I'll keep it together in the same PR with the new features in the future 👍

**Comment by katbow** [comment]
@nelsonic should we hold off on this until we change the content input to the more generalised UI?

**Comment by katbow** [comment]
@nelsonic it's updated now with the single table for stories and tips! You might notice that the schema does not have a `user_id` at the moment. I need to finish the users/login section first then I'll add that in.

File: web/controllers/tip_controller.ex [hunk 1]
```diff
@@ -0,0 +1,15 @@
+defmodule Healthlocker.TipController do
+  use Healthlocker.Web, :controller
+
+  alias Healthlocker.Post
```
Comment (nelsonic) [inline web/controllers/tip_controller.ex]: thanks for updating this. 👍

**Pull request merged**